src/Components/Rich/Card.vue

```diff
@@ -1,6 +1,11 @@
 <template>
     <div class="card" tabindex="0">
-        <img v-if="imageUri" class="card-image" :src="imageUri" :alt="imageTitle || title">
+        <img
+            v-if="imageUri"
+            class="card-image overlay-image"
+            :src="imageUri"
+            :alt="imageTitle || title"
+            @click.stop="$emit('openInOverlay', $event.target)">
```

Comment:
Potresti prevedere già una funzione con una condizione if(true) che viene invocata sull'evento @click.stop ?
Così da far aprire l'overlay solo con quella condizione che in futuro metteremo non sempre true.
```
@click.stop=shouldOpenOverlay
\\altro codice
shouldOpenOverlay(){
if(true)
 $emit(....)
}
```

Reply:
done